notebooks/hive/PyHive.ipynb

```diff
@@ -0,0 +1,726 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Exploration and Reporting with PyHive"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we show how to explore data in Hive and build reports.\n",
+    "\n",
+    "The example uses a dataset of real estate sales in the Sacramento area, which you can download from [here](http://snurran.sics.se/hops/hive/Sacramentorealestatetransactions.csv).\n",
+    "Create a dataset in your project (without README.md) and upload the CSV file in there.\n",
```

Comment:
@SirOibaf replace with "Create a dataset with a name of your choosing, for example `RawData` and upload the CSV file. Make sure the dataset is empty, you will need to delete the auto-generated README.md".